### ISSUE TITLE
Error Code CAIWS001E

### ISSUE THREAD

**Issue by parkerderek** [pspete/psPAS#189]
**Describe the issue**
After successful authentication, I am unable to retrieve a valid session token. See example using **Get-PASSession**.

**Screenshots & Console Output**
![psPAS Error](https://user-images.githubusercontent.com/14941900/61330485-506a3d80-a7e5-11e9-896d-9b73fbfbd21c.png)

```
PS C:\Users\[REDACTED]> New-PASSession -Credential $cred -BaseURI https://pvwaurl.com -type RADIUS -OTP push
PS C:\Users\[REDACTED]> Get-PASSession
Invoke-PASRestMethod : [403] Couldn't retrieve session token from request header.
At line:28 char:13
+ ...   $result = Invoke-PASRestMethod -Uri $URI -Method GET -WebSession $S ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: ({"ErrorCode":"C...quest header."}:ErrorRecord) [Invoke-PASRestMethod], Ex
   ception
    + FullyQualifiedErrorId : CAIWS001E,Invoke-PASRestMethod
```

**Your Environment**

* PowerShell Version: 5.1
* psPAS Version: 3.17
* CyberArk Version: 9.9.5

**Additional context**
I investigated the fix to _Cannot get token session #62_, and the web.config is normal.

**Comment by pspete** [comment]
Only the Classic API will be available in CyberArk Version 9.9.5; `New-PASSession` favors the V10 API by default.

You can try this command which uses parameters specific to the Classic API:
`New-PASSession -Credential $cred -BaseURI https://pvwaurl.com -useRadiusAuthentication $True -OTP 123456`

**Comment by pspete** [comment]
@parkerderek - Thanks for reporting this, I've been able to reproduce.
An auth token value returned when using the v10 API endpoint to authenticate whilst against v9.x is of a different length than `New-PASSession` is expecting. 
This is able to be rectified by a very minor update to the code; change will be making its way into the master branch and the PSGallery shortly.
